diamm/templates/website/archive/archive_list.jinja2

```diff
@@ -1,10 +1,13 @@
 {% extends "base.jinja2" %}
 
 {% block body %}
-    <h1>Archive List</h1>
-    <ul>
-    {% for archive in content.results %}
-        <li><a href="{{ archive.url }}">{{ archive.name }}</a> ({{ archive.city.name }})</li>
-    {% endfor %}
-    </ul>
+    <div class="col-md-12">
```

Comment:
This is bootstrap template code (old). You should use the skeleton template code (see https://github.com/diamm/diamm/blob/master/diamm/templates/website/archive/archive_detail.jinja2 for example)